src/kemal/config.cr

```diff
@@ -55,6 +58,16 @@ module Kemal
       HANDLERS.insert(0, @logger.not_nil!)
     end
 
+    def check_port
+      begin
+        TCPSocket.new(@host_binding, @port).close
+        logger.write "#{@port} port is used by another process. Please select a different port.\n"
```

Comment:
The message is better like this.

`Port 3000 is used by another process. Please specify another port with -p option when starting your Kemal application.`